packages/extension/public/locales/en/translation.json

```
@@ -1,25 +1,35 @@
+  "Disconnect all": "",
+  "Connect all": "",
   "<unknown>": "",
   "Invalid address": "",
   "Total stake": "",
   "Fee": "",
   "Cancel": "",
-  "Insufficient balance to cover transaction fee.": "",
-  "Selected proxy account lacks funds for the fee.": "",
-  "Insufficient balance to complete the transaction.": "",
-  "Insufficient balance for transaction deposit.": "",
+  "Insufficient balance to cover transaction fee. {{extraText}}": "",
```

Biome

[error] The key Insufficient balance to cover transaction fee. {{extraText}} was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)